lib/Runtime/Debug/DiagStackFrame.cpp

```diff
@@ -359,8 +359,7 @@ namespace Js
             varThis = scriptContext->GetLibrary()->GetNull();
         }
 
-        Js::Arguments args(1, (Js::Var*) &varThis);
-        varResult = pfuncScript->CallFunction(args);
+        varResult = CALL_FUNCTION(pfuncScript->GetScriptContext()->GetThreadContext(), pfuncScript, CallInfo(1), varThis);
```

Comment:
>CALL_FUNCTION [](start = 20, length = 13)

Noticed that CallFunction does stack probe but CALL_FUNCTION macro doesn't, will it be safe to not have probe stack in all new code?

Reply:
I believe so, since the number of arguments are well known and small